lib/tasks/db.rake

```diff
@@ -1,75 +1,9 @@
-unless defined?(Rails)
-  task :environment do
-    require './config/application.rb'
-    fail 'ActiveRecord Not Found' unless Module.const_defined?(:ActiveRecord)
-  end
-
-  namespace :db do
-    desc 'Migrate the database through scripts in db/migrate. Target specific version with VERSION=x'
-    task migrate: :environment do
-      ActiveRecord::Migrator.migrate('db/migrate', ENV['VERSION'] ? ENV['VERSION'].to_i : nil)
-      Rake::Task['db:schema:dump'].invoke unless Gris.env.production?
-    end
-
-    desc 'Rollback to a previous migration. Go back multiple steps with STEP=x'
-    task rollback: :environment do
-      ActiveRecord::Migration.verbose = true
-      step = ENV['STEP'] ? ENV['STEP'].to_i : 1
-      ActiveRecord::Migrator.rollback('db/migrate', step)
-    end
-
-    desc 'Create the database'
-    task create: :environment do
-      db = Gris.db_connection_details
-      admin_connection = db.merge(database: 'postgres',
-                                  schema_search_path: 'public')
-      ActiveRecord::Base.establish_connection(admin_connection)
-      begin
-        ActiveRecord::Base.connection.create_database(db['database'])
-      rescue ActiveRecord::StatementInvalid => e
-        if e.message =~ /DuplicateDatabase/
-          $stderr.puts "#{admin_connection['database']} already exists"
-        else
-          raise
-        end
-      end
-    end
+if File.file? 'config/database.yml'
```

Comment:
I don't love how tightly coupled this is at present to having a specific file present. 

Alternatively we could do a couple things:
* Move this out of Gris and into the rakefile for generated apps
* Provide a mechanism to tell during initialization whether Gris is being loaded on its own or via a Gris app